src/components/NoMessageSelected.vue

```diff
@@ -44,6 +44,12 @@ export default {
 		NcEmptyContent,
 		IconMail,
 	},
+
+	computed: {
+		welcomeMessage() {
+			return t('mail', 'Welcome to {cloudName} Mail', { cloudName: window.OC.theme.name })
```

Comment:
```suggestion
			return t('mail', 'Welcome to {cloudName} Mail', { cloudName: window?.OC?.theme?.name ?? 'Nextcloud' })
```

to make it bullet-proof